owners/info_server.js

```diff
@@ -27,12 +27,12 @@ class Server {
    * Constructor.
    *
    * @param {?express.App} app optional Express app to use.
-   * @param {Logger} [logger=console] logging interface.
+   * @param {Logger} logger logging interface.
    * @param {function} initRoutes function adding routes.
    */
-  constructor(app, logger) {
+  constructor(app, logger = console) {
```

Comment:
I wasn't fundamentally opposed to using JSDoc for default args, since that's a valid pattern. However, doing this cleanup and having parity with `amphtml` also SGTM.